Makefile

```diff
@@ -55,6 +55,9 @@ doccheck: generate
 	@cd docs; for doc in *.md; do if [ "$$doc" != "README.md" ] && ! grep -q "$$doc" *.md; then echo "ERROR: No link to documentation file $${doc} detected"; exit 1; fi; done
 	@echo OK
 
+docs:
+	cd "./internal/docsgenerator/" ; go run generatemddocs.go
```

Comment:
Nit: Does just go run and full path not work? Can we skip change directory?

Reply:
@lilic This is changed, now we are running from without changing directory